doc/dune-libs.rst

```diff
@@ -57,7 +57,8 @@ example:
 
   let () =
     C.main ~name:"foo" (fun c ->
-      let has_clock_gettime = C.c_test c clock_gettime_code ~link_flags:["-lrt"] in
+      let has_clock_gettime =
+        C.c_test c clock_gettime_code ~link_flags:["-lrt"] in
```

Comment:
Gratuitous diff.

Reply:
I can remove it — it is just it was not 80 chars wide.

Reply:
OK, that's fine then :)